.github/workflows/cypress-workflow.yml

```diff
@@ -7,14 +7,14 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.8.0'
-  OPENSEARCH_VERSION: '2.8.0-SNAPSHOT'
+  OPENSEARCH_DASHBOARDS_VERSION: '2.9.0'
+  OPENSEARCH_VERSION: '2.9.0-SNAPSHOT'
```

Comment:
should this be 3.0 since its main?

Reply:
There were issues with main backend due to which cypress tests fail to run, will create an issue to get that fixed and revert this to main